elements/lisk-dpos/src/delegates_info.ts

```diff
@@ -160,7 +160,7 @@ export class DelegatesInfo {
 		return true;
 	}
 
-	private async _updateMissedBlocks(
+	private async _updateSafetyMeasure(
```

Comment:
Why this name? I don't see a reference in the LIP but it would be nice if possible to have a name that hints at what this does. It would seem the original name `_updateMissedBlocks` was better.

Reply:
I updated the name because it's no longer only updating missed blocks.
Maybe `_updateProductivity` is better?

Reply:
Oh yes `_updateProductivity` is better